gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/visitor/ExecutionContextBuilderVisitor.java

```diff
@@ -52,4 +55,19 @@ public Void visit(SortAndPaginateNode sortAndPaginateNode) {
   public Void visit(NoOpNode noOpNode) {
     return null;
   }
+
+  @Override
+  public Void visit(SelectionAndFilterNode selectionAndFilterNode) {
+    return null;
```

Comment:
Is thsi fully implemented?

Reply:
Yes. The only relevant node for this visitor is `visit(DataFetcherNode dataFetcherNode)`